docs/_docs/integration-checks-reference.md

```diff
@@ -788,7 +789,7 @@ If your object is returned with a differently named `id` field (such as
 `contact_id`), write code to rename it.
 
 This check is similiar to `D010`. But unlike `D010`, this one validates the live
-polling results in the task history instead of the static samples in your
+polling results in the [Zap History](https://zapier.com/app/history) instead of the static samples in your
```

Comment:
Oh, I didn't realize that! Good to know. I'm happy to do whichever is easiest/quickest.

Reply:
You can go ahead and finish it here and we can get it over to the platform repo. Hopeuflly we won't have to do this twice for too much longer.

FWIW, I use the CLI readme as the "source of truth" for this content, since that's where I spend most of my time.

Reply:
Thanks for the second eye! I'll go ahead and submit the updates I have based on feedback, and keep that in mind in the future. (I spend more time in the UI docs normally :)